### PR TITLE
Edit doc repo in template

### PR DESCRIPTION
**Pull request by hustodemon**
Update documentation templates

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by hustodemon** [comment]
@Loquacity I pushed a commit with the direct link that opens the "new issue" form (with `documentation` label and with the [correct project](https://github.com/SUSE/spacewalk/projects/31) set. Could you review this? If you approve the PR, I'll merge & backport to SUSE Manager.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ After:
 
 ## Documentation
 - No documentation needed: **add explanation. This can't be used if there is a GUI diff**
-- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)
+- [Documentation issue](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31) was created (GitHub automatic link expected below)
```
Comment (paususe) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Is this right?

SUSE/doc-susemanager is public but SUSE/spacewalk is not.

Should this rather be uyuni-project/uyuni-docs ?

Comment (Loquacity) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: Docs track issues in Spacewalk, just like the developers.

Comment (hustodemon) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: @paususe As @Loquacity wrote: the doc issues are tracked like "development" issues (which we also link in the upstream PRs). I agree that this is a bit unhandy for external contributors, so maybe we could rephrase the instructions a bit:

`(For SUSE Manager contributors): [Documentation issue](url) was created.`

Would it make more sense?

From my own experience I can say that not having such url in the template leads to (even more) forgetting to address the documentation.

Comment (Loquacity) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: External contributors can use the "report an issue" link in the Resources menu of the docs, which opens an issue in the Uyuni docs repo. I see them there, and create a duplicate in the Spacewalk repo for us to track. Either way, we have a process to handle them.

Comment (paususe) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: @hustodemon 

What about something like this?
``
Documentation issue for [Uyuni](https://github.com/uyuni-project/uyuni-docs/issues/new) or [SUSE Manager](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31) was created (GitHub automatic link expected below)
```

Comment (hustodemon) [reply .github/PULL_REQUEST_TEMPLATE.md]: > External contributors can use the "report an issue" link in the Resources menu...

But the purpose of the link in the template is to suggest the contributors to create a doc issue together with their PR. They don't know, that they need to go to the Resources menu.

> What about something like this? Documentation issue for [Uyuni]...

I think that stating "issue for uyuni OR issue for suma" is a bit confusing: for a SUMA devel, it's not absolutely clear whether they should create suma or uyuni issues. I made this clear in my latest commit: suma devels create suma issues, community devels create uyuni issues.

**Pull request merged**